experimental/host/mt7688/gpio.go

```diff
@@ -10,53 +10,53 @@ import (
 
 // All the pins supported by the CPU.
 var (
-	GPIO0  *Pin // I2S_SDI
-	GPIO1  *Pin // I2S_SDO
+	GPIO0  *Pin // I2S_DIN
+	GPIO1  *Pin // I2S_DOUT
 	GPIO2  *Pin // I2S_WS
-	GPIO3  *Pin // I2S_CLK
-	GPIO4  *Pin // I2C_SCLK
-	GPIO5  *Pin // I2C_SD
-	GPIO6  *Pin // SPI_CS1
+	GPIO3  *Pin // I2S_SCK
+	GPIO4  *Pin // I2C_SCL
+	GPIO5  *Pin // I2C_SDA
+	GPIO6  *Pin // SPI_CS1, CLK0
```

Comment:
Use `SPI0_CS0`. Some CPUs have multiple SPI buses. I expect this one to only have one so I presume it's SPI0. Being specific is good.

Updated the ones below the same way, `SPI0_CLK`, `SPI0_MOSI`, etc.